packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -163,12 +140,11 @@ class UnThemedLogRow extends PureComponent<Props, State> {
       theme,
       getFieldLinks,
     } = this.props;
-    const { showDetails, showContext, hasHoverBackground } = this.state;
+    const { showDetails, showContext } = this.state;
     const style = getLogRowStyles(theme, row.logLevel);
     const styles = getStyles(theme);
     const { errorMessage, hasError } = checkLogsError(row);
     const logRowBackground = cx(style.logsRow, {
-      [styles.hoverBackground]: hasHoverBackground,
       [styles.errorLogRow]: hasError,
```

Comment:
Did not understanmd this hasError & errorLowRow, the styles set textWeak, but could not find any row where this was used (when searching & expanding error log rows) . But I left it here as there might be some special case for here.

Reply:
See https://github.com/grafana/grafana/pull/28359 for more context.